Execution Result for predict_anomaly_scores.py
--------------------------------------------------
=== Execution successful (attempt 1/3) in 11.89s ===
LOG_START:DATA_LOADED
Rows: 105830, Columns: 6
LOG_END:DATA_LOADED
Checking whether there is an H2O instance running at http://localhost:54321..... not found.
Attempting to start a local H2O server...
  Java Version: openjdk version "21.0.8" 2025-07-15; OpenJDK Runtime Environment (build 21.0.8+9-Debian-1); OpenJDK 64-Bit Server VM (build 21.0.8+9-Debian-1, mixed mode, sharing)
  Starting server from /usr/local/lib/python3.11/site-packages/h2o/backend/bin/h2o.jar
  Ice root: /tmp/tmppcd_heui
  JVM stdout: /tmp/tmppcd_heui/h2o_unknownUser_started_from_python.out
  JVM stderr: /tmp/tmppcd_heui/h2o_unknownUser_started_from_python.err
  Server is running at http://127.0.0.1:54321
Connecting to H2O server at http://127.0.0.1:54321 ... successful.
Warning: Your H2O cluster version is (5 months and 12 days) old.  There may be a newer version available.
Please download and install the latest version from: https://h2o-release.s3.amazonaws.com/h2o/latest_stable.html
--------------------------  ----------------------------------
H2O_cluster_uptime:         01 secs
H2O_cluster_timezone:       Etc/UTC
H2O_data_parsing_timezone:  UTC
H2O_cluster_version:        3.46.0.7
H2O_cluster_version_age:    5 months and 12 days
H2O_cluster_name:           H2O_from_python_unknownUser_0zxkdc
H2O_cluster_total_nodes:    1
H2O_cluster_free_memory:    3.984 Gb
H2O_cluster_total_cores:    16
H2O_cluster_allowed_cores:  16
H2O_cluster_status:         locked, healthy
H2O_connection_url:         http://127.0.0.1:54321
H2O_connection_proxy:       {"http": null, "https": null}
H2O_internal_security:      False
Python_version:             3.11.13 final
--------------------------  ----------------------------------
Parse progress: |████████████████████████████████████████████████████████████████| (done) 100%
LOG_START:MODEL_LOADED
Model: IsolationForest_model_python_1757359255519_1
Algorithm: H2OIsolationForestEstimator
LOG_END:MODEL_LOADED
isolationforest prediction progress: |/usr/local/lib/python3.11/site-packages/h2o/job.py:81: UserWarning: Test/Validation dataset column 'vendedor_id' has levels not trained on: ["520", "531", "535", "611", "655", "660", "663", "700", "726", "753", "758", "836", "852", "917", "921", "930", "934"]
  warnings.warn(w)
/usr/local/lib/python3.11/site-packages/h2o/job.py:81: UserWarning: Test/Validation dataset column 'cliente_nombre' has levels not trained on: [" Agustina Da Rosa", " Ana Graciela Kamrat PaszkovicZ", " DELPIAZZO ABOGADOS SC", " Dahiana Oribe", " Fabricio Gonzalez", " Jessica Silveira", " Juan Manuel Tapia", " Paola Genova", " Ririro Sas", "(NO USAR)DODILEDFER SRL", ...3592 not listed..., "valentina marquez", "vanessa acosta", "veikel lopez ", "vera moran", "virginia martino", "yohana franco ", "Álvaro Rostan", "Ángeles González", "ÑENDY SA", "Óptica Brun"]
  warnings.warn(w)
/usr/local/lib/python3.11/site-packages/h2o/job.py:81: UserWarning: Test/Validation dataset column 'invoice_prefix' has levels not trained on: ["A-10007", "A-10008", "A-10018", "A-10026", "A-1003", "A-10031", "A-10045", "A-10049", "A-10051", "A-10060", ...17961 not listed..., "Deposito de Ch en Abitab22", "Deposito de Ch en Abitab3", "Deposito de Ch en Abitab4", "Deposito de Ch en Abitab5", "Deposito de Ch en Abitab6", "Deposito de Ch en Abitab7", "FC", "RET", "RINV", "RNT"]
  warnings.warn(w)
/usr/local/lib/python3.11/site-packages/h2o/job.py:81: UserWarning: Test/Validation dataset column 'invoice_name_length' has levels not trained on: ["24"]
  warnings.warn(w)
/usr/local/lib/python3.11/site-packages/h2o/frame.py:1983: H2ODependencyWarning: Converting H2O frame to pandas dataframe using single-thread.  For faster conversion using multi-thread, install polars and pyarrow and use it as pandas_df = h2o_df.as_data_frame(use_multi_thread=True)

  warnings.warn("Converting H2O frame to pandas dataframe using single-thread.  For faster conversion using"
███████████████████████████████████████████| (done) 100%
PREDICTIONS_FILE_START:/workspace/pipeline_3ee8ca9e-5834-4f88-a64d-e4a2ec2834dc/predictions.csv:PREDICTIONS_FILE_END
LOG_START:PREDICTION_SUMMARY
{"total_predictions": 105830, "anomalies_detected": 12336, "anomaly_percentage": 11.656430123783426, "mean_anomaly_score": 0.15329893106043532, "min_anomaly_score": 0.0, "max_anomaly_score": 1.1083916083916083}
LOG_END:PREDICTION_SUMMARY
H2O session _sid_be4e closed.


[File Persistence Status]
✗ forecast_plot.png (missing)
✓ predictions.csv (10074962 bytes)